packages/@webex/plugin-meetings/src/breakouts/README.md

```diff
@@ -119,8 +119,11 @@ breakout.remove(participant)
 // This can only be done when breakouts are inactive
 breakout.delete()
 
-// Create a breakout session
-breakouts.create(sessionName)
+// Create breakout sessions
+breakouts.create(sessions)
```

Comment:
we need a better description here on what a session is i.e. the argument type of sessions

Reply:
Do we know what options are available here?

Reply:
As locus design:
"sessions" : [ {
      "name" : "Breakout Session 1",
      "subConfId" : 1,
      "anyoneCanJoin" : false,
      "id" : "ea2db6e1-2da9-4f72-9d62-f099e5e3c7b4"
    }, {
      "name" : "Breakout Session 2",
      "anyoneCanJoin" : true
    } ]